java/mtsj/core/src/main/java/io/oasp/application/mtsj/bookingmanagement/service/impl/rest/BookingmanagementRestServiceImpl.java

```diff
@@ -13,11 +14,13 @@
 import io.oasp.application.mtsj.bookingmanagement.logic.api.to.TableSearchCriteriaTo;
 import io.oasp.application.mtsj.bookingmanagement.service.api.rest.BookingmanagementRestService;
 import io.oasp.module.jpa.common.api.to.PaginatedListTo;
+import org.springframework.validation.annotation.Validated;
 
 /**
  * The service implementation for REST calls in order to execute the logic of component {@link Bookingmanagement}.
  */
 @Named("BookingmanagementRestService")
+@Validated
```

Comment:
We finally updated the validation documentation. Thus, validation should be done on the logic layer due to the fact, that you can simultaneously validate batch operations as well. Otherwise, you have to implement it again.